### ISSUE TITLE
downloads not available on german appstores

### ISSUE THREAD

**Issue by tiiiecherle** [anurodhp/Monal#119]
Hey,

first of all thanks for this project. I would really love to try it, but both apps (macos and ios) are not available in the world-wide appstore and are not shown in germany.

It would be nice to have them available and additionally the macos version via download on the website.

Thanks in advance

Kind regards

**Comment by maufl** [comment]
If I understand correctly, the Push Notification service would be the only service you would provide to EU users? If you don't log any IP addresses or user identifiers, this should not be a problem at all. Could the crash analytics just be disable for EU users? Or simply opt in? Then the GDPR would not apply to you, right?

**Comment by eun-ice** [comment]
The problem is, it’s a new and complex law and and probably has neither time nor the will to navigate it. The law poses next to no risk compared to many laws in the US, e.g. software patents, copy right laws and so on. It’s just that gdpr is new and there is still a lot of FUD around. It will take time.

**Comment by muppeth** [comment]
But @anurodhp you are not collecting any information yourself right? As far as i can see its a xmpp client that works with peoples own accounts. Push services are integrated to the devices and users agree to apple push in that case, so as far as i know, you have nothing to do with gdpr as you do not collect any data yourself.

**Comment by anurodhp** [comment]
Yes there’s very little incidental data like ip that get colllected in http log files. It’s not sufficient that I’m not collecting anything I need to do the paperwork to prove I am not and respond to gdpr requests appropriately to be compliant.

**Comment by muppeth** [comment]
But you are serving only a client. So its up to server providers to be gdpr complient. you are not offering a server so there is nothing to be gdpr complient. 

Apple users consent to push notification by using ios, and its apple in this case that needs to be gdpr complient.

**Comment by muppeth** [comment]
Also in case you are concerned about push and apple. Gdpr clearly says that collecting data if its necessary for the service / application to work is permitted. Just like google analytics, mailchimp etc is still legal in europe, so is your application (which to my understanding does not collect anything nor process anything on your server).

There is really nothing you should be worry about. You do not collect nor process any data on any server you own, which means you have no data to for example provide to people upon request, or delete any data since you dont have any and have no access to any data of any user using your application.

**Comment by muppeth** [comment]
aahhhh ok i see. I did not know he needs to run his own push. This is also very easy to implement. All you need to do is add privacy policy (which you should have no matter the gdpr), where you specify, what is that you are collecting and if you are processing, note what you do with the data and for how long do you store it. You could have a look at this privacy policy https://framagit.org/hubzilla/core/blob/dev/doc/gdpr1.md

**Comment by da2x** [comment]
@anurodhp still available in the Norwegian App Store. Norway is not an EU member state, but is part of the European Economic Area (EEA; EU member states plus Iceland, Liechtenstein, and Norway.) The GDPR also applies in the EEA as of 2018-07-20.

That being said: just make the push service opt-in.

**Comment by da2x** [comment]
—have you considered the impact of the [California Consumer Privacy Act](https://www.caprivacy.org/about)? Should pull the app from Californians too.

**Comment by agru** [comment]
Anything new about this? I'd really like to see monal in the German app store!
ChatSecure is also using it's own push service! Why is it possible there, and not with Monal?
XMPP is used by more and more people here in Germany, at least as I can see this. And I would appreciate to see a second product for iOS, besides of ChatSecure.
